### DISCUSSION TITLE
Automatically setting a new team lead if a team lead leaves the company

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#5655] [Ideas]
A [user ](https://app.hubspot.com/contacts/3888472/contact/12181701) 🔒 wrote in to support asking how he can take over as team lead for a team where the team leader left the company. The user who wrote in is also the organization's billing leader. 

If a team lead leaves perhaps we can automatically assign the billing leader as the new team lead? Otherwise we create a situation where no one at the company can do account management tasks for that team. 

Also, in order to set a new team lead we have to make manual changes to our DB. A mutation to automatically assign a new team lead seems simpler. 

[Relevant Slack discussion with @BartoszJarocki ](https://parabol.slack.com/archives/C4JAUUZ9P/p1636554509049200) 🔒

**Comment by ackernaut** [comment]
How would we know if the team lead has left the company?

Does it make sense to allow a billing leader to reassign team leads?

What happens if the billing leader leaves?

Are there common practices that other services use that allow for changing account ownership when they get stuck on the problem of admins leaving?

**Comment by drewhousman1010** [comment]
A user at a company that recently signed an enterprise deal pointed out that one of his teams had no team lead. 

This happened because their old team lead went inactive.

[Slack thread with details](https://parabol.slack.com/archives/C4JAUUZ9P/p1644278107341589) 🔒

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/46023501) 🔒  writes: 

> Hello. Have a question about team leader. We have a new team leader on the team. The old one no longer works. Is it possible to change a lead's team without deleting the existing team and the entire history of that team?

**Comment by enriquesanchez** [comment]
Related: [Allow for more than one Team Lead per Team](https://github.com/ParabolInc/parabol/discussions/6105)

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/16028251) 🔒 wrote in to support because they could not remove users from their team. The team in question had no billing leader assigned:

> I don't see any billing leader in our "Organization settings -> Members"... I'm assuming that is because we're on the free tier ?
> 
> There are members showing on that page that are not showing up on the "Manage Team" page and it's those I would need to remove.

**Comment by drewhousman1010** [comment]
A [user ](https://app.hubspot.com/contacts/3888472/contact/12374401) 🔒 says: 

> My team lead left the company and I had trouble administering my account. I couldn't remove anyone.

**Comment by drewhousman1010** [comment]
A[ user ](https://app.hubspot.com/contacts/3888472/contact/10414801) 🔒 writes: 

> how can someone be promoted to a Team Lead if in the Team is no Team Lead left? we have multiple those teams in our org

**Comment by enriquesanchez** [comment]
A [user](https://app.hubspot.com/live-messages/3888472/inbox/3252044438#email) 🔒 writes:

>Hi! I inherited a parabol team but can't seem to remove people that left the organization.
It look like I'm not declared "Team Lead". The problem is that the actual Team Lead has now left. How can we solve that?

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/67465801) 🔒 asks if there is a way to re-assign the team lead if they leave the company.
